docs/apache-airflow/authoring-and-scheduling/deferring.rst

```diff
@@ -198,14 +198,76 @@ When your operator resumes, Airflow adds a ``context`` object and an ``event`` o
 
 If your operator returns from either its first ``execute()`` method when it's new, or a subsequent method specified by ``method_name``, it will be considered complete and finish executing.
 
-You can set ``method_name`` to ``execute`` if you want your operator to have one entrypoint, but it must also accept ``event`` as an optional keyword argument.
```

Comment:
I moved this comment into the section on deferring multiple times as it is demonstrated there.